commands/common.go

```diff
@@ -194,15 +189,18 @@ func (c *config) RemoveTenet(name string, group string) error {
 		return errors.Trace(err)
 	}
 	var tenets []TenetConfig
-	for _, t := range c.AllTenets() {
+	err = errors.Errorf("tenet %q not found", name)
+	for _, t := range g.Tenets {
 		if t.Name != name {
 			tenets = append(tenets, t)
```

Comment:
I think it would be clearer if we returned within the if (and set g.Tenets = tenets there). Then just return an error after the for loop. A few less lines of code that way.